### ISSUE TITLE
Only enabled allowed Images for selection

### ISSUE THREAD

**Issue by jideabdqudus** [upload-io/react-uploader#4]
Hello <Maintainers>

I took react-uploader for a spin and works well so far for me, although I found a little issue with using it with my current application. Not much of a bug but would allow me to take full advantage of the library 

Whenever an upload is going to be made, I've added the mime to be images ALONE as follows 

```mimeTypes: ["image/jpeg", "image/png", "image/jpg"]```

But then, when user clicks on the upload button and the file directory opens up the user can select other file types before getting an error, compared to another library I tested, only allowed file types are available for selection. This feature would help in improving the user experience generally. 

This is an example of when I try to upload with 'react-uploader' - the user is allowed to select even a doc file for image upload 
<img width="652" alt="Screenshot 2023-03-22 at 21 39 56" src="https://user-images.githubusercontent.com/45945474/227033298-7f6cc877-ff70-40e6-b905-1325c55d75fa.png">

The expected result should be this, with the file types not allowed disabled:
<img width="673" alt="Screenshot 2023-03-22 at 21 40 24" src="https://user-images.githubusercontent.com/45945474/227033496-9ebe4cb3-2434-443e-a012-35992456b9fd.png">

In Html, this could be done by:

```
<label>Image File
  <input type="file" name="myImage" accept="image/png, image/gif, image/jpeg" />
</label>
```

If there's an extra prop, or I'm missing something I'd be glad to check it out again. Thank you!

**Comment by ljwagerfield** [comment]
Hi @jideabdqudus 👋 

Excellent suggestion, thank you for raising!

Fixed in `react-uploader@3.11.0`. 

Please re-open if you have any issues re. this ticket 👍

**Comment by jideabdqudus** [comment]
Fantastic, thanks for the quick update @ljwagerfield  👍🏾
